src/Features/Core/Portable/EditAndContinue/ManagedEditAndContinueCapability.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    /// <summary>
+    /// The capabilities that the runtime has with respect to edit and continue
+    /// </summary>
+    [Flags]
+    internal enum ManagedEditAndContinueCapability
```

Comment:
>ManagedEditAndContinueCapability [](http://example.com/codeflow?start=18&length=32)

Let's rename to EditAndContinueCapabilities (plural)- the "Managed" prefix is only used in the debugger contracts since debugger supports both native nad managed EnC. Roslyn only implements managed, so it's not necessary to restate that.